packages/jest-core/src/SearchSource.ts

```diff
@@ -37,7 +37,7 @@ export type TestSelectionConfig = {
 };
 
 const globsToMatcher = (globs: Array<Config.Glob>) => (path: Config.Path) =>
-  micromatchSome(replacePathSepForGlob(path), globs, {dot: true});
+  !!micromatch([replacePathSepForGlob(path)], globs, {dot: true}).length;
```

Comment:
same `length` thing here and other places. I at least find it more readable.

(https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md would have potentially been nice, but it requires type info so we can't use it (yet?))